plugins/inputs/systemd_units/systemd_units.go

```diff
@@ -128,109 +119,80 @@ var subMap = map[string]int{
 	"elapsed": 0x00a0,
 }
 
+// SystemdUnits is a telegraf plugin to gather systemd unit status
+type SystemdUnits struct {
+	Timeout       config.Duration
```

Comment:
Please also add the toml tag here
```suggestion
	Timeout       config.Duration `toml:"timeout"`
```